packages/ladle/lib/cli/vite-plugin/parse/get-entry-data.js

```diff
@@ -14,24 +14,26 @@ const debug = debugFactory("ladle:vite");
 
 /**
  * @param {string[]} entries
+ * @param {boolean} hasPackages
  */
-export const getEntryData = async (entries) => {
+export const getEntryData = async (entries, hasPackages) => {
```

Comment:
It wasn't entirely obvious what this param was doing
```suggestion
export const getEntryData = async (entries, { hasPackages = false }) => {
```